src/less/tooltip/base/tooltip.less

```diff
@@ -162,6 +162,66 @@ span.tourtip {
     width: @icon-width;
 }
 
+.dialog-infotip[role="dialog"] {
+    .dialog-base();
+}
+
+.dialog-infotip__window {
+    .dialog-window();
+
+    padding-bottom: 15vh;
+    top: 15%;
```

Comment:
This is getting overriden by the `top:50` in the the other styles, is it needed?